requirements.txt

```diff
@@ -1,7 +1,7 @@
 autopep8
 backports.unittest-mock
 click
-docker[ssh]>=3.6.0
+docker[ssh]==3.7.0
```

Comment:
docker and docker-compose versions are bond. Please use range version instead of one version.
Besides requirements.txt is only used in test. The dependency specified used in packaging stage is setup.py. Please also change that file.

Reply:
Updated